src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/models/nfs.fsal.ts

```diff
@@ -1,5 +1,9 @@
+export enum SUPPORTED_FSAL {
+  CEPH = 'CEPH',
```

Comment:
it should ideally be FS or CEPHFS.

CEPH FSAL doesn't make sense. 

I know this was introduced earlier and not in PR but it looks confusing.

Reply:
I think this is the way its defined in backend as well https://docs.ceph.com/en/latest/cephfs/nfs/#configuring-nfs-ganesha-to-export-cephfs thats why we have it like that.